Improved formatting of INSTALL.md

SufianBabri opened the pull request:
This PR has some readability improvements to the installation readme file:
- typo in "build directory"
- adjusted heading sizes for better readability

**I have one question though:**
While installing this theme, I faced an issue similar to #2 and it took me some time to navigate around it. Will it be a good idea to add a section like "common issues" with possible solutions to INSTALL.md file?

jnsh commented:
Thank you for the typo correction and the suggestion. I can see the purpose of this, but the numbering seems a bit excessive addition to me. Maybe just increasing the size of the headers would be enough, since that greatly improves the size contrast? Or adding the numbering only for the actual build steps (getting the source, dependencies, configure, build)?

> **I have one question though:**
> While installing this theme, I faced an issue similar to #2 and it took me some time to navigate around it. Will it be a good idea to add a section like "common issues" with possible solutions to INSTALL.md file?

The INSTALL.md is already rather long and complex, so I'd prefer not to add any additional sections to it. I understand that the build failing from undetermined cinnamon/gnome-shell/gtk3 version is a common struggle when you compile the theme for the first time, but I've already tried to be clear about it on the [versioned themes](https://github.com/jnsh/arc-theme/blob/master/INSTALL.md#versioned-themes) section. I suppose this might be worth mentioning at the [Common problems](https://github.com/jnsh/arc-theme/wiki/Common-problems) wiki page though. I'd certainly like to make this clearer, if you have any additional suggestions.

SufianBabri commented:
@jnsh I have removed the numbering and adjusted the headings for better readability (similar to the format of README.md).

The INSTALL.md could also have a link to the Wiki page's "Common problems" page (for quicker reference). Should I do it as well?

jnsh commented:
Thanks, the updated revision looks good to me.

> The INSTALL.md could also have a link to the Wiki page's "Common problems" page (for quicker reference). Should I do it as well?

Currently there's nothing related to build/installation in the wiki page, and even if there was, the INSTALL.md is the main build documentation that explains everything, so there shouldn't be a need to look for the wiki page. The common problems is mainly meant for a quick reference for non-build related issues, that don't have an obvious solution, and is already mentioned on README.md.
